### DISCUSSION TITLE
A difference between restoring from existing backup and using Data Branching feature.

### DISCUSSION THREAD

**Discussion by devunt** [planetscale/discussion#432] [Q&A] [answered]
After reading the [documentation](https://planetscale.com/docs/concepts/data-branching) about PlanetScale's Data Branching feature, I wonder what is the difference between using Data Branching feature and restoring from backup by creating new branch.

What I understand is that they are exactly the same, as Data Branching also uses the latest backup to seed the initial data.

If they are the same, why Data Branching is only available on Team plan while restoring is available on every plan?
I need the Data Branching feature, but I wonder is it worth to upgrade the plan just to use Data Branching feature or should I just stick to manual restoring process. If they are the same there's no reason to upgrade the plan as I'm currently on Scalar and my cap is enough.

**Comment by jgreet** [comment, marked as answer]
Hi @devunt, 

I don't think you should upgrade to a Team plan _solely_ to enable the Data Branching feature in the web application. The way it works currently, restoring the most recent backup of the branch manually will get you the same result.
